android_automation/page_action/home_page.py

```diff
@@ -0,0 +1,236 @@
+from time import sleep
+from appium.webdriver.common.appiumby import AppiumBy
+from selenium.common import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from com_utils.api_control import home_banner_info
+from com_utils.element_control import aal, aalc, aals, scroll_control, swipe_control
+from ios_automation.page_action import context_change
+from ios_automation.page_action.bottom_sheet import close_bottom_sheet
+
+
+def check_home_logo(wd):
+    try:
+        aal(wd, 'navi_logo_btn')
+        print('HOME 탭으로 이동 확인')
+    except NoSuchElementException:
+        print('HOME 탭으로 이동 확인 실패')
+
+
+def click_search_btn(wd):
+    aalc(wd, 'com.the29cm.app29cm:id/imgSearch')
+    print("상단 검색 아이콘 선택")
+    sleep(2)
+
+
+def click_close_life_tab(wd):
+    # 디폴트 선택 화면 확인
+    top_tabs = aal(wd, 'com.the29cm.app29cm:id/tabs')
+    culture_tab = aal(top_tabs, '//android.view.View[@index=4]/android.widget.TextView')
+    if culture_tab.text == '푸드':
+        print("푸드 탭 존재")
+        aalc(wd, 'life_tab')
+        print("라이프 탭 해제")
+    else:
+        print('라이프 탭 디폴트 아님')
+
+    sleep(3)
+
+
+# click_tab_name : 선택하려는 상단 탭 이름 입력
+def click_tab_name(wd, click_tab_name):
+    aalc(wd, click_tab_name)
+    print(f'{click_tab_name} 탭 선택')
+    sleep(2)
+
+
+def save_tab_names(wd):
+    tab = aals(wd, f'//*[contains(@label, "라이프")]/../../../XCUIElementTypeCell')
+    tab_name_list = []
+    for text in tab:
+        tab_name = text.find_element(AppiumBy.XPATH, '//XCUIElementTypeStaticText').text
```

Comment:
```suggestion
        tab_name = aal(text, '//XCUIElementTypeStaticText').text
```
위 코드 요렇게 사용할 수 있을 것 같은데, 다른 코드 구조화 할 때 활용해보면 좋을 것 같습니다.

Reply:
넵 해당 코드도 ios에 있던거라 다음 pr에 제거하도록하겠습니다